conduct.md

```diff
@@ -40,25 +40,34 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+Students are encouraged to:
+
+- Be proactive about communicating.
```

Comment:
Please provide a description of how students can be proactive. Ergo, utilizing Discord to message TL/Professor/Team about any changes/concerns.